packages/env/src/index.ts

```diff
@@ -37,7 +37,7 @@ if (environment !== 'production') {
       WEBSOCKET_REFRESH_SECRET_TOKEN_KEY: 'refresh-refresh-token-key',
       WORKERS_WEBSOCKET_SECRET_TOKEN: 'workers-secret-token',
       DRIVE_DISK: 'local',
-      DEFAULT_PROVIDER_ID: '1',
+      DEFAULT_PROVIDER_ID: '10000000000000000000', // Crazy number to avoid conflicts
```

Comment:
What about -1?

Reply:
Not sure if it's a good idea to pass to postgresql a -1